admin_manual/release_notes.rst

```diff
@@ -74,7 +74,8 @@ The Password Policy Extension
 .. note::
    These features don't apply to users imported from LDAP or other backends but only for local users created by administrators or the `Guests <https://marketplace.owncloud.com/apps/guests>`_ extension.
 
-Imposing password expiration and history policies enhance security. For example, when users are forced to choose a new password, they can be blocked from using their previous passwords (the password history count is configurable). However, increasing security can also decrease user satisfaction, such as forcing them to change their password upon first login, or change their passwords too frequently (such as every 90 days). That said, we encourage an acceptable balance between security and usability.
+Imposing password expiration and history policies enhance security. For example, when users are forced to choose a new password, they can be blocked from using their previous passwords (the password history count is configurable). To ensure a good experience it is possible to inform users via email and internal notifications (web interface and clients) a configurable amount of days before the password expires and when the password has expired.
```

Comment:
- "enhance" => "enhances"
- "blocked" => "prevented" ? sounds better
- in general from my little experience with tech writing, using statements in parenthesis isn't recommended as it breaks the flow (ok to leave them for now)
- not sure if it's worth to say it's configurable here. you could add a sentence at the end saying what is configurable if you think it's important
- "internal notifications (...)" => "internal notifications that appear in web interface and clients." ?
- I suggest moving the last part into a separate sentence: "It is possible to configure the number of days before the password expires and when the notification is sent".

Reply:
I incorporated your changes and restructured the text, too. Please check again.